server/src/main/java/org/apache/iotdb/db/metadata/lastCache/LastCacheManager.java

```diff
@@ -222,7 +222,7 @@ public static long getLastTimeStamp(IMeasurementMNode node, QueryContext queryCo
       try {
         QueryDataSource dataSource =
             QueryResourceManager.getInstance()
-                .getQueryDataSource(node.getPartialPath(), queryContext, null);
+                .getQueryDataSource(node.getPartialPath(), queryContext, null, false);
```

Comment:
Add a comment, `true` or `false` both ok here, because LastPointerReader will do itself sort logic instead of depending on `fillOrderIndex`.